example/README.md

```diff
@@ -232,7 +232,7 @@ packer and [atlas](http://atlas.hashicorp.com).
 # the monorail_rack setup script.
 
 # deployment variables
-pxe_count=1
+PXE_COUNT=1
 ```
 
 Changing the number of `pxe_count` within the running configuration script will
```

Comment:
could you also help update 'pxe_count' to 'PXE_COUNT' in line 231 and 238 's annotation